eng/scripts/CodeChecks.ps1

```diff
@@ -101,7 +98,11 @@ try {
         if ($LastExitCode -ne 0) {
             $status = git status -s | Out-String
             $status = $status -replace "`n","`n    "
-            LogError "Generated code is not up to date. You may need to run eng\scripts\Update-Snippets.ps1 or sdk\storage\generate.ps1 or eng\scripts\Export-API.ps1"
+            LogError "Generated code is not up to date.`n" + `
```

Comment:
don't know about `sdk\storage\generate.ps1` but making sure it is not longer needed

Reply:
It's no longer needed.